### ISSUE TITLE
Feature request: Reorder the selected item from the History window option.

### ISSUE THREAD

**Issue by dragon122** [acoppes/unity-history-window#50]
Currently, only selecting from the Hierarchy triggers reordering in the History window, it would be nice to have this behavior when selecting items from the History window directly as well.

**Comment by acoppes** [comment]
Like if you select something from the bottom of the history it goes up to the be the first element? (or vice versa if you have it inverted)? If it is that, the idea to not do this initially was to not scroll you off where you are when you click on something, but maybe could be an option so if you want that behavior it is your choice.

**Comment by dragon122** [comment]
> Like if you select something from the bottom of the history it goes up to the be the first element? (or vice versa if you have it inverted)? If it is that, the idea to not do this initially was to not scroll you off where you are when you click on something, but maybe could be an option so if you want that behavior it is your choice.

Yes, I think it's useful because the most used elements (but not useful enough to add them to favorites) will eventually go way down (/up, if inverted), History Window can completely replace the Hierarchy in my workflow, meaning that the reordering will never be triggered again, unless I specifically ping + select in the Hierarchy needed item, which is annoying.
